specs/gloas/p2p-interface.md

```diff
@@ -301,7 +301,7 @@ The following validations are removed:
 | `BELLATRIX_FORK_VERSION` | `bellatrix.SignedBeaconBlock` |
 | `CAPELLA_FORK_VERSION`   | `capella.SignedBeaconBlock`   |
 | `DENEB_FORK_VERSION`     | `deneb.SignedBeaconBlock`     |
-| `EIP7732_FORK_VERSION`   | `eip7732.SignedBeaconBlock`   |
+| `GLOAS_FORK_VERSION`     | `gloas.SignedBeaconBlock`     |
```

Comment:
Do we need Electra fork version? Also just curious, why Fulu doesn't have this section?